menuboot.html

```diff
@@ -0,0 +1,129 @@
+<!doctype html>
+<html lang="en">
+  <head>
```

Comment:
15 errors when using an html validator. You should use one on all your pages - https://validator.w3.org/nu/